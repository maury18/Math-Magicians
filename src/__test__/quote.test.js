import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/quote';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot', () => {
    const domTree = renderer.create(<Quote />).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
