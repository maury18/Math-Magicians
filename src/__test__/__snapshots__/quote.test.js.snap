// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<div
  className="quote-container"
>
  <p>
    <span
      className="theQuote"
    >
      Mathematics is not about numbers equations, computations, or algorithms. It is about undersatanding.
    </span>
    <span
      className="theAuthor"
    >
      -William paul Thurston
    </span>
  </p>
</div>
`;
